app/views/home/pricing.html.erb

```diff
@@ -1,2 +1,37 @@
-<h1>Home#pricing</h1>
-<p>Find me in app/views/home/pricing.html.erb</p>
+<section class="pricing-header bg-light px-3 py-3 pt-md-5 pb-md-4 mx-auto text-center">
+  <h1 class="display-4">Plans</h1>
+  <h2 class="text-muted">Get started now on opeNode!</h2>
+  <p class="lead">We provide a free sandbox to get started and you will not have to enter your credit card information.<br/>We accept both Paypal and Cryptocurrency (Bitcoin, Bitcoin Cash, etc.) payments.</p>
+</section>
+<section class="pt-8 pt-md-11 bg-dark p-3 p-md-5">
+  <div class="container">    
+    <div class="card-deck mb-3 text-center">
+      <div class="row">
+      <% YAML.load_file("#{Rails.root}/config/pricing.yml").map { |o|o.deep_symbolize_keys }.each do |pricing| %>
+        <div class="col-3 mb-5">
+        <div class="card border-success shadow-sm ml-0 mr-0 h-100 hvr-grow">
+          <div class="card-header bg-success text-white">
+            <h4 class="my-0 font-weight-normal"><%= pricing[:name] %></h4>
+          </div>
+          <div class="card-body">
+            <h1 class="card-title pricing-card-title"><%= pricing[:title] %></h1>
+            <h3><small class="text-muted"><%= pricing[:subtitle] %></small></h3>
+            <ul class="list-unstyled mt-3 mb-4">
+              <% pricing[:features].each do |feature|  %>
+                <li><%= feature %></li>
```

Comment:
would be better if features are aligned together (not centered) 

![image](https://user-images.githubusercontent.com/8007722/71545874-88fa6f80-295e-11ea-8105-0f0514a2d150.png)

Reply:
same issue @gastonsoto

Reply:
Added lines and more spacing between list elements.
![Screen Shot 2019-12-30 at 15 22 54](https://user-images.githubusercontent.com/786208/71594867-56947200-2b18-11ea-9415-51817183d8e8.png)

Reply:
looks good!